packages/twenty-server/src/engine/metadata-modules/field-metadata/interfaces/field-metadata-settings.interface.ts

```diff
@@ -10,6 +10,10 @@ type FieldMetadataDefaultSettings = {
   isForeignKey?: boolean;
 };
 
+type FieldMetadataLinksSettings = {
+  isDomain?: boolean;
```

Comment:
Do we want to introduce this in this PR? I don't see where this is used

Reply:
Right it's not actually